lib/go-parser/go-binary.ts

```diff
@@ -18,8 +18,14 @@ export class GoBinary {
     const pclnTab = goElfBinary.body.sections.find(
       (section) => section.name === ".gopclntab",
     );
+
+    // some CGo built binaries might not contain a pclnTab, which means we
+    // cannot scan the files.
+    // TODO: from a technical perspective, it would be enough to only report the
+    // modules, as the only remediation path is to upgrade a full module
+    // anyways. From a product perspective, it's not clear (yet).
     if (pclnTab === undefined) {
-      throw Error("no pcln table found in Go binary");
+      throw Error(`no pcln table present in Go binary`);
```

Comment:
just nitpicking, but why did you use back-ticks instead of quotes? (I also wonder why the linter was ok with it)

Reply:
No reason actually, why? Is there a difference? 😅

Reply:
Back-ticks are usually used as [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). They can be used as ordinary strings, but for the sake of code uniformity we use one type of quotes. This is why the formatter _will_ replace single quotes with double quotes, even though they're both valid in JS

Reply:
Ah okay, that makes perfect sense.
I adjusted the quotes now.